apps/sms/views/shared/js/settings.js

```diff
@@ -51,9 +56,13 @@ var Settings = {
     this._serviceIds = [];
 
     if (!settings) {
+      this.sendReadReport = Promise.resolve(false);
```

Comment:
nit: I think you can put this directly on the object, after `smsServiceId` in line 35, and then remove this line.